alarms/models/overdrive_checkouts_alarms.py

```diff
@@ -20,7 +20,7 @@ def run_checks(self):
         self.logger.info("\nOVERDRIVE CHECKOUTS\n")
         try:
             overdrive_count = self.overdrive_client.get_count(self.yesterday)
-        except OverDriveWebScraperError:
+        except:
```

Comment:
It might be good to print out the exception in the logs, up to you though

Reply:
Good idea!